src/pages/summer/Login/LoginInput/LoginInput.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+
+import './LoginInput.scss';
+
+class LoginInput extends React.Component {
+  state = {
+    idInput: '',
+    pwInput: '',
+    idVld: false,
```

Comment:
```suggestion
    idValid: false,
```
이 경우 애매한 약어보다는 풀네임을 쓰는게 더 좋을 것 같습니다